CHANGELOG.md

```diff
@@ -9,8 +9,23 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+### Changed
+
+### Deprecated
+
+### Removed
+
+### Fixed
+
+## [1.10.0]
```

Comment:
The links at the bottom need updated.

Reply:
It looks like 1.9.0 is also missing a link.

Reply:
Ha, I totally didn't see the bottom.  I was wondering how these links were working.  TIL!

Reply:
Done in ed6f64c.